controller/api/public/grpc_server.go

```diff
@@ -467,6 +474,10 @@ func processRequestRate(
 
 		datapoints := make([]*pb.MetricDatapoint, 0)
 		for _, value := range metric.Values {
+			if value.Value == 0 {
+				continue
+			}
```

Comment:
This change makes the request count metric behave the same as the success rate metric. Specifically, we will no longer return a datapoint if request count == 0; it's already the case that we don't return a success rate datapoint when request count == 0. We should address how to handle sparse timeseries in a subsequent pull request.